test/test-tcp-read-stop-start.c

```diff
@@ -0,0 +1,136 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
Use the copyright from here: https://github.com/libuv/libuv/blob/v1.x/test/test-ipc-heavy-traffic-deadlock-bug.c